README.md

```diff
@@ -23,6 +26,7 @@ Contains crates specific to our application. Can depend on libraries located in
 
 - Install `rustup`: `curl https://sh.rustup.rs -sSf | sh`
 - Install `cargo-make`: `cargo install cargo-make`
+- Run `cargo make` in the root folder of the repository, this will install various crates & tools such as rustfmt & clippy
```

Comment:
🔧 If I'm not wrong, this will fail here if build dependencies have not been installed (e.g. Docker, nvm, yarn). How about moving this 3 lines down (after `Install 'nvm'`) and add something like:

```suggestion
- Install additional build dependencies `libzmq3-dev,` (*Note: the name may vary depending on your OS). 
- Run `cargo make` in the root folder of the repository, this will install various crates & tools such as rustfmt & clippy
```